build.gradle.kts

```diff
@@ -217,6 +217,7 @@ dependencies {
     "sqplugins"("org.sonarsource.iac:sonar-iac-plugin:1.16.0.3845")
     if (artifactoryUsername.isNotEmpty() && artifactoryPassword.isNotEmpty()) {
         "sqplugins"("com.sonarsource.cpp:sonar-cfamily-plugin:6.46.0.62229")
+        "sqplugins"("com.sonarsource.plsql:sonar-plsql-plugin:3.9.0.5181")
```

Comment:
I thought the goal was to only support PL/SQL in connected mode? If yes, then we should not embed it.